WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -1100,3 +1102,12 @@ extension AbstractPostListViewController: NetworkAwareUI {
         return tableViewHandler.resultsController.isEmpty()
     }
 }
+
+extension AbstractPostListViewController: NetworkStatusDelegate {
+    func networdStatusDidChange(active: Bool) {
+        // The network status is received from a background thread, only in this view controller
```

Comment:
I'm curious, why is this controller different?

Reply:
I am really curious as well 😊 I spent just a bit of time trying to figure out what makes it different, and I couldn't find it. I will take another look today